lib/stacker/stack.rb

```diff
@@ -141,8 +160,77 @@ def update options = {}
       end
     end
 
+    def describe_change_set
+      retries = 6
+      changes = []
+      while changes.empty?
+        resp = region.client.describe_change_set(
+          change_set_name: change_set,
+          stack_name: name
+        )
+        changes = resp.changes
+        if changes.empty?
+          raise CannotDescribeChangeSet.new 'Empty change set' if retries == 0
+          retries -= 1
+          sleep 1
+        end
+      end
+      changes.map do |c|
+        rc = c.resource_change
+        {
+          type: c.type,
+          change: {
+            logical_resource_id: rc.logical_resource_id,
+            action: rc.action,
+            replacement: rc.replacement,
+          }
+        }
+      end
+    end
+    memoize :describe_change_set
+
+    def pretty_change_set
+      riw = describe_change_set.map do |c|
+        c[:change][:logical_resource_id].length
+      end.max
+      fmt = "%-6s %-#{riw}s %-5s"
```

Comment:
> This is color formatting, ya? 

It's basically table formatting, setting field widths.

> Maybe we could have an object

Sure. I'm just in get-it-done mode and have left my "Any clean-up after the SDK update" TODO unchecked.